chia/consensus/default_constants.py

```diff
@@ -50,6 +50,7 @@
     "BLOCKS_CACHE_SIZE": 4608 + (128 * 4),
     "WEIGHT_PROOF_RECENT_BLOCKS": 1000,
     "MAX_BLOCK_COUNT_PER_REQUESTS": 32,  # Allow up to 32 blocks per request
+    "RUST_CONDITION_CHECKER": 662000 + 100000,
```

Comment:
@Yostra since it has taken a while for this PR to be approved, I suppose we should bump this some